migration_sqlite_mysql.rst

```diff
@@ -0,0 +1,45 @@
+.. _migration-sqlite-mysql:
+
+===================================
+Migrating data from SQLite to MySQL
+===================================
+
+Storage Service supports both SQLite and MySQL databases. In most cases, it is
+preferably to use the latter since SQLite does not support concurrent writes.
+This is exarcerbated by the fact that Storage Service performs long write
+transactions.
+
+.. note:
+
+   Check out `issue #952`_ for more details.
+
+Please find below an overview of the steps needed to move your data to MySQL.
+Make sure that you take a back up of your SQLite database before you proceed.
+
+1. Stop Storage Service::
+
+    systemctl stop archivematica-storage-service
+
+2. Export your data to JSON::
+
+    su - archivematica -s /bin/bash -c "set -a -e -x; source /etc/default/archivematica-storage-service; cd /usr/lib/archivematica/storage-service/; /usr/share/archivematica/virtualenvs/archivematica-storage-service/bin/python manage.py --exclude auth.permission --exclude contenttypes --exclude=sessions.session --natural-foreign --natural-primary --indent 4 dumpdata > /tmp/datadump.json"
+
+3. Create the MySQL database::
```

Comment:
Since it's a new database, should we mention that privileges should be granted to the `username` below?